build.py

```diff
@@ -2,9 +2,15 @@
 # -*- coding: utf-8 -*-
 
 from cpt.packager import ConanMultiPackager
+import os
 
 if __name__ == "__main__":
-	builder = ConanMultiPackager(username='bincrafters');
-	
+	builder = ConanMultiPackager(username='bincrafters',
```

Comment:
this also seems to be redundant

Reply:
weird, no parameters and environment variables should be necessary at all
it works for all recipes with no issues
I'll check CI settings and let you know what's going wrong